pyfzz/pyfzz.py

```diff
@@ -20,4 +20,4 @@ def read_file(self, filename) -> List[Tuple[str, List[int]]]:
     def write_file(self, filename, data: List[Tuple[int, int, int]]):
         with open(filename, 'w') as f:
             for bar in data:
-                f.write(f'{bar[0]} {bar[1]} {bar[2]}\n')
\ No newline at end of file
+                f.write(f'{bar[2]} {bar[0]} {bar[1]}\n')
```

Comment:
I have two other major issues: (1) I see that you included phat source codes directly into our repo. While it seems okay to do so under the github licenses, I think the best thing to do would be to download phat from their own repo each time when you try to install fzz. I don't know whether this can be done, but it's the most 'beautiful' ways to do so. If you really want to directly copy their codes to our repo, we may want to check the details of the license or email the authors (but I really recommend we choose the other way). (2) This can be done later -- but I think it's best you merge the five commits into one commit to make the history cleaner.

Reply:
For the phat issue: By talking to users, I believe people are looking for a library that can be easily installed. The reason I am including the phat headers to remove external dependencies. I agree the way you mentioned is the most elegant way but in general people do not prefer to download multiple repos separately. I think phat also has BSD clause and since yours is also a OSI, I think it is covered within the license.
For issue (2): I agree with you. The commit history can be made clear and I think I can merge the commits. For now, I am closing the PR. I will open a new draft PR.

Reply:
Okay, if the license issue is okay, then I'm okay with copying the phat codes here. (Maybe we should mention it somewhere but I will do that later)